src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.FrameworkReferenceResolution.targets

```diff
@@ -369,6 +369,12 @@ Copyright (c) .NET Foundation. All rights reserved.
 
     </ResolveTargetingPackAssets>
 
+    <PropertyGroup>
+      <!-- Unescape this semicolon-delimited ordered list so that it's passed as a list to targets
+           that consume it. -->
+      <PackageConflictPreferredPackages>$([MSBuild]::Unescape($(PackageConflictPreferredPackages)))])</PackageConflictPreferredPackages>
```

Comment:
Initially I was thinking to use an intermediate property to do this, to avoid squashing a user-provided property, but I guess since it will always overwrite a user property then we don't need to do that.  This is fine an minimal.

Reply:
I figured this was ok since it overwrites a property that was just overwritten (as the output from the `ResolveTargetingPackAssets` task).